docs/blog/2023-06-27-jupyterlab-myst.md

```diff
@@ -0,0 +1,117 @@
+
+When you have installed the extension open any notebook or markdown document, and the markdown will be rendered as MyST. For example, showing the rich frontmatter as well as an admonition in dark mode.
+
+<figure>
+<video src="https://github.com/executablebooks/meta/raw/d30cae55226c762a956c5a33cf9fd6e93557f3f8/docs/_static/videos/jupyterlab-markup-dark.mp4" autoplay muted webkit-playsinline="true" playsinline loop style="max-width:100%"></video>
+<figcaption>Showing MyST Markdown rendering in dark mode!</figcaption>
+</figure>
+
+Some of the new additions that we added for v2 of the extension are:
+
+- GitHub link previews for issues, pull-requests and code ([demo](https://mystmd.org/guide/external-references#issues-and-pull-requests))
+- Wikipedia integrations to show hover-cards ([demo](https://mystmd.org/guide/external-references#wikipedia-links))
+- Task lists that edit your markdown cells ([demo](https://mystmd.org/guide/quickstart-jupyter-lab-myst#task-lists))
+- Rabbit-hole links for cross-referenced content ([demo](https://mystmd.org/guide/quickstart-myst-markdown#links-cross-references))
+- Glossaries and terms ([demo](https://mystmd.org/guide/glossaries-and-terms))
+- Support for proofs ([demo](https://mystmd.org/guide/proofs-and-theorems)) and exercises ([demo](https://mystmd.org/guide/exercises))
+- Copy buttons on your presentational code ([demo](https://mystmd.org/guide/code#code-blocks))
+
+These all work in your notebook cells as well as now opening a markdown document in preview.
+
+<figure>
+<video src="https://github.com/executablebooks/meta/raw/d30cae55226c762a956c5a33cf9fd6e93557f3f8/docs/_static/videos/jupyterlab-markdown-preview.mp4" autoplay muted webkit-playsinline="true" playsinline loop style="max-width:100%"></video>
+<figcaption>Instant MyST Markdown preview for full markdown documents.</figcaption>
+</figure>
+
+## Inline Variables & Execution
+
+JupyterLab MyST allows you to create and evaluate inline expressions using the `{eval}` role. These turn your markdown cells into places that you can quickly evaluate a simple expression, such as:
+
+- The value of the variable `x`: `` {eval}`x` ``
+- Expand a sympy equation `polynomial`: `` {eval}`expand(polynomial)` ``
+- Add inline matplotlib “sparklines”
+- Add ipywidgets directly inline for sliders and other dropdowns
+
+<figure>
+<video src="https://github.com/executablebooks/meta/raw/d30cae55226c762a956c5a33cf9fd6e93557f3f8/docs/_static/videos/jupyterlab-myst-inline.mp4" autoplay muted webkit-playsinline="true" playsinline loop style="max-width:100%"></video>
+<figcaption>Inline execution in markdown cells using the <code>{eval}</code> role.</figcaption>
+</figure>
+
+Inline expressions open up a whole new world for communicating with interactive computation, and we are excited to explore this and other integrations with [thebe](https://github.com/executablebooks/thebe) and [JupyterLite](https://jupyterlite.readthedocs.io/en/latest/) in the coming months.
```

Comment:
...open up a whole new world [of narrative flow] for communicating with interactive computation...

Reply:
Based on your comment @stevejpurves:

```suggestion
Inline expressions open up a whole new world for communicating with interactive computation directly in your markdown narrative, and we are excited to explore this and other integrations with [thebe](https://github.com/executablebooks/thebe) and [JupyterLite](https://jupyterlite.readthedocs.io/en/latest/) in the coming months.
```